Source/ACE.Database/CharacterDatabase.cs

```diff
@@ -374,6 +428,17 @@ public void SaveCharacterOptions(Character character)
             ExecutePreparedStatement(CharacterPreparedStatement.CharacterOptionsUpdate, character.CharacterOptions.GetCharacterOptions1Flag(), character.CharacterOptions.GetCharacterOptions2Flag(), character.Id);
         }
 
+        /// <summary>
+        /// This function will save all of the CharacterPositions 
+        /// </summary>
+        public void SaveCharacterPositions(Character character)
```

Comment:
I'm pretty sure we can just do 1-off saves when the player changes them.  The only save that should be saved with the character is the physical position and maybe the last used portal.  We can save the others when they tie/bind.